File: vinicius012/programacao_basica/Questao05.java
import java.util.Scanner;

public class Questao05 {
     public static void main(String[] args) {
     Scanner s = new Scanner(System.in);
     
     System.out.print("Quatidade de contratos simples: ");
     int x = s.nextInt();
     System.out.print("Quantidade de contratos completos: ");
     int y = s.nextInt();      
     
     float z = 2300;
     
     float a = 3500; 

     float de = x * z * 0.1f;
     
     float da = y * a * 0.1f;

     System.out.println(" Lucro obtido: $" + (de + da)); 
   }
}

File: vinicius012/programacao_basica/questao05.py
print("**  Seguradora xyz  **")
print("     >> Sistema de avaliação <<  ")
q = 2300
w = 3500
x = int(input("Qtd de contratos simples: "))
y = int(input("Qtd de contratos completos: "))
resul0 = x * q * 0.1
resul1 = w * y * 0.1
total = resul0 + resul1
print((" Lucro Total: R$") + str(total))
